src/pages/Login/Login.test.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+import AuthProvider from '../../providers/Auth';
+import Login from './index';
+
+describe('Login Components', () => {
+    test('User name input is present', () => {
+        render(
+            <AuthProvider>
+                {' '}
```

Comment:
why is this magic empty string necessary? Can it be avoided? Actually, is this scenario necessary?